LICENSE

```diff
@@ -28,7 +28,6 @@ The Chapel implementation is composed of two categories of code:
 
    third-party/
      chpl-venv        python packages supporting chpldoc & tests
-       subprocess32   Back port of python3 subprocess to python2   Python
```

Comment:
Note that we also have this page: https://chapel-lang.org/license.html

IMO, that page should just be a link to this file.